torchvision/transforms/transforms.py

```diff
@@ -261,6 +261,77 @@ def __repr__(self):
         return self.__class__.__name__ + '()'
 
 
+class RandomTransforms(object):
+    """Base class for a list of transformations with randomness
+
+    Args:
+        transforms (list or tuple): list of transformations
+    """
+
+    def __init__(self, transforms):
+        assert isinstance(transforms, (list, tuple))
+        self.transforms = transforms
+
+    def __call__(self, *args, **kwargs):
+        raise NotImplementedError()
+
+    def __repr__(self):
+        format_string = self.__class__.__name__ + '('
+        for t in self.transforms:
+            format_string += '\n'
+            format_string += '    {0}'.format(t)
+        format_string += '\n)'
+        return format_string
+
+
+class RandomApply(RandomTransforms):
+    """Apply randomly a list of transformations with a probability
+
+    Args:
+        transforms (list or tuple): list of transformations
+        proba (float): probability
```

Comment:
can you maybe make it `p`, so that it is consistent with other transforms?

Reply:
Totally agree